apps/web/lib/i18n/en.ts

```diff
@@ -167,11 +167,11 @@ export const en = {
 			INPUT_INVITE_CODE: 'Input invitation code.',
 			INPUT_INVITE_CODE_DESC: 'Enter the invitation code we sent to your email.',
 			INVALID_INVITE_CODE_MESSAGE: 'Invalid Code',
-			WELCOME_TEAMS: 'Welcome to Ever Teams',
+			WELCOME_TEAMS: 'Real-Time Clarity, Real-Time Reality™.',
 
-			COVER_TITLE: 'Follow your teams work progress in real-time!',
+			COVER_TITLE: 'Open Work  and Project Management Platform',
```

Comment:
Why did you put 2 spaces here before "and"?

Reply:
Fixed